### ISSUE TITLE
Show negative word count in red.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [emuikernel/wittytwitter#53]
```
visual cues for user to know that they passed the 140 char-count limit.
```

Original issue reported on code.google.com by `alan...@gmail.com` on 22 Nov 2007 at 1:11

**Comment by GoogleCodeExporter** [comment]
```
hi, I think i can finish this feature.
```

Original comment by `yeqiwei` on 5 Jan 2008 at 3:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `alan...@gmail.com` on 7 Feb 2008 at 6:07
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
I've got this one completed in a set of enhancements that I'll commit once the 
clickonce that fixes #145/login issue goes live.
```

Original comment by `jfol...@gmail.com` on 12 Jul 2008 at 7:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `jfol...@gmail.com` on 13 Jul 2008 at 3:26
- Changed state: **Fixed**
